.gitlab-ci.yml

```diff
@@ -270,7 +270,11 @@ system_probe-build:
     - inv -e deps
   tags: [ "runner:main", "size:large" ]
   script:
-    - inv -e system-probe.build
+    # TODO(sami): removeme go1.10.1
+    - curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
```

Comment:
This works, but it would probably be preferable to just use a builder image with the right go version in it... You'd probably then be able to forget about all the gimme changes in the invoke tasks. Having those builders could come handy in the future to quickly just swap over, etc. 

ie. `486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/ebpf:1.10.1`

Reply:
👍